reconcile/utils/glitchtip/models.py

```diff
@@ -65,17 +74,21 @@ class Project(BaseModel):
     teams: list[Team] = []
 
     @root_validator
-    def slugify(cls, values):  # pylint: disable=no-self-argument
-        values["slug"] = values.get("slug") or slugify(values.get("name"))
+    def slugify(  # pylint: disable=no-self-argument
+        cls, values: dict[str, Any]
```

Comment:
Can `dict` inputs become `MutableMapping` as per [dev-guidelines](https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#abstract-collection-types)?

Reply:
I haven't had time, but I'd really like to see if there's a plugin that would remind us of this. I forget to do this for `dict` more than any other use case (compared to say `Iterable`).